### ISSUE TITLE
p2p-external-port out of range

### ISSUE THREAD

**Issue by polar-it** [turtlecoin/turtlecoin#889]
Seems there is some missed check on possibile allowable ports used for deamons nat settings.
Is possibile to set p2p-external-port on out of range port, example :3419769.
The port (  3419769 ) is out of 0-65535.

Currently checking the on-line peers list, some peers have port out-of-range.

**Comment by brandonlehmann** [comment]
Confirmed

```
"142.44.242.106:16789113",
"5.186.205.243:1060473",
"58.185.155.109:3419769",
```

**Comment by brandonlehmann** [comment]
Resolved in #890
